info/v1/container.go

```diff
@@ -848,6 +848,25 @@ type PerfStat struct {
 	Cpu int `json:"cpu"`
 }
 
+type MemoryBandwidthMonitoringStats struct {
+	// The 'mbm_total_bytes'.
+	TotalBytes uint64 `json:"mbm_total_bytes,omitempty"`
+
+	// The 'mbm_local_bytes'.
+	LocalBytes uint64 `json:"mbm_local_bytes,omitempty"`
+}
+
+type CacheMonitoringTechnologyStats struct {
```

Comment:
Maybe just CacheStats?

Reply:
See: https://01.org/cache-monitoring-technology

Reply:
Exported struct should have a descriptive comment starting with CacheMonitoringTechnologyStats, please add link to description